### ISSUE TITLE
[NSR]

### ISSUE THREAD

**Issue by carolinalisboa** [GLTEN/terms-issues#1]
## Synonym name
(please give the synonym name and language)
yield component
## Concept URI
(please provide the URI for the concept the synonym should be added to)
http://agrovoc.uniroma2.it/agrovoc/agrovoc/en/page/?uri=http%3A%2F%2Faims.fao.org%2Faos%2Fagrovoc%2Fc_24419

**Comment by carolinalisboa** [comment]
Maybe it could be grain quality (which is there already). I am checking this info with the requester (Lionel) 
https://www.ebi.ac.uk/ols/ontologies/co_340/terms?iri=http://www.cropontology.org/rdf/CO_340:Quality%2520traits

**Comment by richardostler** [comment]
I spoke with Lionel the other day at coffee and I think yield component is legitimate - it covers a range of different measures made associated with yield. The alternative is to define each thing that is measured.

**Comment by carolinalisboa** [comment]
Yes, that is it indeed. This definition can be broader (as Lionel mentioned)or narrowed, probably this decision will hang on the level of details of the GLTEN-scheme will capture, right?

**Comment by richardostler** [comment]
No this is a decision for the person entering the data. The portal allows the user to enter both levels of details i.e. either yield component or several different measures of yield
